frontend/src/app/_services/menu.service.ts

```diff
@@ -33,7 +33,7 @@ export class MenuService {
     currentNodeSelector.path = path;
     currentNodeSelector.open = true;
     currentNodeSelector.children = apiResult.children.map(child => this.buildHierarchyNodeSelector(child, `${path}_${child.slugName}`));
-    currentNodeSelector.toBeDisplayed = true;
+    currentNodeSelector.toBeDisplayed = false;
```

Comment:
If `currentNodeSelector.toBeDisplayed = false;` then you can delete this line as it's the default value